packages/flutter/test/widgets/nested_scroll_view_test.dart

```diff
@@ -676,6 +676,13 @@ void main() {
     await gesture.up();
     debugDefaultTargetPlatformOverride = null;
   });
+
+
+  testWidgets('NestedScrollView and SliverOverlapAbsorber for PR #48947', (WidgetTester tester) async {
```

Comment:
Can you use a test name that is more descriptive of the bug rather than the PR number?